### PR TITLE
add a quick blog post on the OpenPGPSummit ...

### PR DESCRIPTION
**Pull request by hpk42**

### CODE REVIEW

**Comment by r10s** [comment]
looks good to me, thanks for all the insights, holger.

wondering if there were there really two patrick or if this was a copy+paste error or so.

btw. when merging: might be that the post is not immediately visible as the 22.10. is in the future from the view of netlify (had this before and was really searching for the cause ...). not sure which timezone is used there, to get around this problem, the date can be modified to 2018-10-21. or we just wait some hours...

File: _posts/2018-10-22-openpgpsummit.md [hunk 1]
```diff
@@ -0,0 +1,90 @@
...
+or "g." who does crypto-research for 25 years, has broken 
+a lot of systems and has a very thorough yet refreshingly 
+pragmatic stance on crypto/security questions ... he appreciated
+and positively commented a lot of the concepts we are trying to
+employ with DeltaChat ...
+
+To be fair, am only reporting here on a part of what i encountered ...
+apart from three plenary talks on saturday, there were 20-30 one-hour
+sessions and countless informal ones ... but as my train is finally
+entering the black forest and it's long past midnight i am windining down
```
Comment (ralphtheninja) [inline _posts/2018-10-22-openpgpsummit.md]: winding down

Comment (ralphtheninja) [inline _posts/2018-10-22-openpgpsummit.md]: winding down

Comment (ralphtheninja) [inline _posts/2018-10-22-openpgpsummit.md]: winding down

Comment (ralphtheninja) [inline _posts/2018-10-22-openpgpsummit.md]: winding down

**Pull request merged**